src/ServiceControl/ServiceControl.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\ServiceControl.Transports.Msmq\ServiceControl.Transports.Msmq.csproj" />
```

Comment:
is this intentional?

Reply:
I could do it as a separate PR, but it fixes the F5 experience

Reply:
Ok. I have no problem with this change as long as does not affect the packaging etc.

Reply:
I don't know anything about SC packaging, but this change causes the msmq dlls to appear in the bin folder.

Reply:
Hmm, I thought the other transports were removed during packaging, but that's only in the acceptance tests. I'll revert this commit for now and come up with a batter way to solve this